src/test/java/teammates/test/cases/pagedata/StudentFeedbackResultsPageDataTest.java

```diff
@@ -96,19 +96,33 @@ public void testAll() throws EntityDoesNotExistException {
         assertEquals("", questionBundle1.getQuestionDetails().getAdditionalInfo());
         assertEquals("", questionBundle2.getQuestionDetails().getAdditionalInfo());
 
-        assertNotNull(questionBundle1.getResponseTables());
-        assertNotNull(questionBundle2.getResponseTables());
+        ______TS("giver & receiver in self/others response tables");
 
-        assertEquals("You", questionBundle1.getResponseTables().get(0).getRecipientName());
+        assertNotNull(questionBundle1.getSelfResponseTables());
+        assertNotNull(questionBundle1.getOthersResponseTables());
+        assertNotNull(questionBundle2.getSelfResponseTables());
+        assertNotNull(questionBundle2.getOthersResponseTables());
 
-        assertNotNull(questionBundle1.getResponseTables().get(0).getResponses());
-
-        assertEquals("You", questionBundle1.getResponseTables().get(0).getResponses()
+        assertTrue(questionBundle1.getSelfResponseTables().get(0).isGiverNameYou());
```

Comment:
You can always add some comment to state what u are testing (like the one u explained to @joanneong )

A example here `// test that self response table contains correct response`